CHANGELOG.md

```diff
@@ -5,6 +5,133 @@ All notable changes to this project will be documented in this file.
+
+### Tropical Geometry
+
+- [#4838](https://github.com/oscar-system/Oscar.jl/pull/4838) Fix `tropical_variety_zerodimensional`
+- [#4781](https://github.com/oscar-system/Oscar.jl/pull/4781) Add `roots` for tropical polynomials
+- [#4703](https://github.com/oscar-system/Oscar.jl/pull/4703) Add tropical prevarieties generated by intersecting tropical hypersurfaces
+- [#4697](https://github.com/oscar-system/Oscar.jl/pull/4697) Remove broken tropical Groebner basis shortcut for binomial ideals
+- [#4447](https://github.com/oscar-system/Oscar.jl/pull/4447) Add `positive_tropical_variety`
+- [#4445](https://github.com/oscar-system/Oscar.jl/pull/4445) Add tropical linear spaces from graphs
+
+### Changes related to serializing data in the MRDI file format
+
+- [#4331](https://github.com/oscar-system/Oscar.jl/pull/4331) Unify type encoding for similar types
+
+### Only changes experimental parts of OSCAR
+
+- [#4899](https://github.com/oscar-system/Oscar.jl/pull/4899) IntersectionTheory: Rename `abstract_projective_bundle` -> `projective_bundle` and `abstract_flag_bundle` -> `flag_bundle`
+- [#4845](https://github.com/oscar-system/Oscar.jl/pull/4845) GroebnerWalk: Remove perturbed walk
+- [#4772](https://github.com/oscar-system/Oscar.jl/pull/4772) Intersection theory: Introduce Gromov-Witten invariants
+- [#4769](https://github.com/oscar-system/Oscar.jl/pull/4769) Intersection theory: extend documentation on Bott formula
+- [#4764](https://github.com/oscar-system/Oscar.jl/pull/4764) Intersection Theory: Kontsevich spaces
+- [#2183](https://github.com/oscar-system/Oscar.jl/pull/2183) Add basics for quantum analogs
+
+### Improvements or additions to documentation
+
+- [#4758](https://github.com/oscar-system/Oscar.jl/pull/4758) Collapse docstrings in documentation to allow for easier navigation
+
+### Changes related to the package AlgebraicSolving
+
+- [#4894](https://github.com/oscar-system/Oscar.jl/pull/4894) Bump AlgebraicSolving to v0.9.0
+
+### Changes related to the package Nemo
+
+- [#4645](https://github.com/oscar-system/Oscar.jl/pull/4645) Update Nemo to 0.49
+
+### Changes related to the package Singular
+
+- [#4613](https://github.com/oscar-system/Oscar.jl/pull/4613) Optimize conversions from/to Singular
+
+### Other changes
+
+- [#4850](https://github.com/oscar-system/Oscar.jl/pull/4850) Add `quo` for `LaurentMPolyWrapRing`
+- [#4780](https://github.com/oscar-system/Oscar.jl/pull/4780) Oscar Worker Pool and parallel functions functionality such as pmap.
+
+### **TODO** release notes: to be added
+
+If there are any PRs listed below, check their title and labels.
+When done, change their label to "release notes: use title".
+
+- [#4895](https://github.com/oscar-system/Oscar.jl/pull/4895) Bump AA to v0.45, Nemo to v0.50, Hecke to v0.36
+- [#4844](https://github.com/oscar-system/Oscar.jl/pull/4844) [FTheoryTools] Some minor improvements
+- [#4162](https://github.com/oscar-system/Oscar.jl/pull/4162) Serialization Refactor
```

Comment:
```suggestion
```
I added suggestions above with dedicated release notes for these three PRs (taken from the respective discussions)